applications/nrf5340_audio/doc/configuration.rst

```diff
@@ -7,153 +7,173 @@ Configuring the nRF5340 Audio applications
 
 In the default configuration, the gateway application uses USB as the audio source.
 The :ref:`nrf53_audio_app_building` and the testing steps also refer to using the USB serial connection.
 
-To switch to using the 3.5 mm jack analog input, set the :ref:`CONFIG_AUDIO_SOURCE_I2S<nrf53_audio_app_kconfigs>` Kconfig option to ``y`` in the :file:`applications/nrf5340_audio/prj.conf` file for the debug version and in the :file:`applications/nrf5340_audio/prj_release.conf` file for the release version.
+The audio source selection affects the firmware architecture and available features.
+USB audio source is limited to unidirectional streams due to CPU load considerations, while I2S supports bidirectional communication.
+
+.. _nrf53_audio_app_configuration_select_i2s:
+
+Selecting the analog jack input using I2S
+=========================================
+
+To switch to using the 3.5 mm jack analog input, set the :option:`CONFIG_AUDIO_SOURCE_I2S` Kconfig option to ``y`` in the :file:`applications/nrf5340_audio/prj.conf` file for the debug version and in the :file:`applications/nrf5340_audio/prj_release.conf` file for the release version.
 
 When testing the application, an additional audio jack cable is required to use I2S.
 Use this cable to connect the audio source (PC) to the analog **LINE IN** on the development kit.
 
-.. _nrf53_audio_app_adding_FEM_support:
+.. _nrf53_audio_codec_configuration:
 
-Adding FEM support
+Configuring codecs
 ******************
 
-You can add support for the nRF21540 front-end module (FEM) to the following nRF5340 Audio applications:
+The nRF5340 Audio application supports both software and hardware codecs.
+See :ref:`nrf53_audio_app_overview_architecture` for information about how codecs are integrated into the firmware architecture.
 
-* :ref:`Broadcast source <nrf53_audio_broadcast_source_app>`
-* :ref:`Unicast client <nrf53_audio_unicast_client_app>`
-* :ref:`Unicast server <nrf53_audio_unicast_server_app>`
+You can configure the software codec using the following Kconfig options:
 
-The :ref:`broadcast sink application <nrf53_audio_broadcast_sink_app>` does not need FEM support as it only receives data.
+* :option:`CONFIG_SW_CODEC_LC3` - LC3 codec (mandatory for LE Audio)
+* :option:`CONFIG_SW_CODEC_NONE` - No software codec
 
-Adding FEM support happens when :ref:`nrf53_audio_app_building`.
-You can use one of the following options, depending on how you decide to build the application:
+You can configure the hardware codec using the following Kconfig option:
 
-* If you opt for :ref:`nrf53_audio_app_building_script`, add the ``--nrf21540`` to the script's building command.
-* If you opt for :ref:`nrf53_audio_app_building_standard`, add the ``-Dnrf5340_audio_SHIELD=nrf21540ek -Dipc_radio_SHIELD=nrf21540ek`` to the ``west build`` command.
-  For example:
+* :option:`CONFIG_NRF5340_AUDIO_CS47L63_DRIVER` - CS47L63 hardware codec driver
```

Comment:
This is not configuring the HW codec, it is just enabling it, same with the SW codec